lib/engine/game/g_1894/game.rb

```diff
@@ -308,16 +308,19 @@ def setup
           paris_tiles = @all_tiles.select { |t| PARIS_TILES.include?(t.name) }
           paris_tiles.each { |t| t.add_reservation!(plm, 0) }
 
+          remove_random_teleport_company
```

Comment:
With this function call moved to before the `sort_by { rand }` on line 315, this breaks 41 of 42 games in my local copy of the database. We can pin all the games if you want, but this seems like a large unintended side effect for just improving log output.

Reply:
Thanks, for letting me know, I changed that.